stakingModel/test/BasicCalculation.test.js

```diff
@@ -12,144 +12,131 @@ contract("InterestDistribution - Basic calculations", ([user1]) => {
     
   });
 
-  it("Should return correct Avg Yield Rate Per Token", async () => {
+  it("Should return correct cumulative Yield Rate", async () => {
 
-      // 1st stake will not affect AvgYieldRatePerToken and GlobalYieldPerToken because globalTotalStaked and totalStaked both are 0.
+      // 1st stake will not affect GDYieldRate and GlobalYieldPerToken because globalTotalStaked is 0.
       await interestDistribution
-        .stakeCalculation(user1, web3.utils.toWei("20", "ether"), 0, web3.utils.toWei("1", "ether"), web3.utils.toWei("1", "ether"))
+        .stake(user1, web3.utils.toWei("20", "ether"), 0)
         .catch(e => e);
 
+      
+      await interestDistribution
+        .setITokenToTokenRate(web3.utils.toWei("0.4", "ether"))
```

Comment:
comment what does this do

Reply:
done

Reply:
the comment still doesnt explain, the comment should give more information than the method name
setITokenToTokenRate

1. what rate?
2. what do ToToken mean
3. why are you updating the rate here

comments should be descriptive enough so a new developer understand what  it does without digging into the code

Reply:
Added comments